### DISCUSSION TITLE
Newbie question: Multiple WLAN's on multiple interfaces (Fritzbox)

### DISCUSSION THREAD

**Discussion by SiggiKeller** [nzymedefense/nzyme#513] [Questions] [answered]
Hello,

I'm trying to setup nzyme on a Raspi. 
In my home I use a Fritzbox with 2 GHz and 5 GHz frequency and I have two SSID - one for "internal", one for guests. On top of that I have a WLAN repeater in mesh mode. So I end up with four different MAC addresses and eight WLAN's in the list, I would like to monitor.
How do I configure this in the nzyme config file? Currently I could enter the "internal" WLAN with the four MAC addresses and several fingerprints. So nzyme recognizes the "internal" WLAN as monitored networks. The same way I configured the guest WLAN's, but nzyme doesn't monitor them. And as a result I'm getting tons of alarms.

Rgds,
Siggi

**Comment by torstenandrecht** [comment]
I have exactly the same scenario. Where did you found the fingerprints within the Fritz box interface?

**Reply by SiggiKeller** [reply]
The fingerprints can be found in nzyme in the network section.

**Comment by torstenandrecht** [comment, marked as answer]
...start with the internal and analyze afterwards the alerts. based on the alert you need to adjust your configuration. I did it that way during the last 2 days. next step now: e-mail Notification
that article helped me a lot:
https://www.heise.de/select/ct/2021/19/2120113483519321666
